geo_to_hca/utils/handle_errors.py

```diff
@@ -42,3 +43,13 @@ def __str__(self):
         return (f"\nStatus code of the request: {self.response.status_code}.\n"
                 f"Error as returned by ENA:\n{self.error}"
                 f"The provided project title or name was:\n{self.title}\n\n")
+
+
+def no_related_study_err(geo_accession):
```

Comment:
we can add 'annotation' later here and in the similar cases

Reply:
Which annotation are you referring to? Can you please give an example?